app/src/main/java/com/hedvig/app/feature/home/ui/changeaddress/ITPB.kt

```diff
@@ -0,0 +1,105 @@
+package com.hedvig.app.feature.home.ui.changeaddress
+
```

Comment:
Since this composable can be used outside of the changeaddress-package, would it make sense to have some kind of `ui-composable`-package were we put all general composables?

Reply:
Agreed. This component is made to be generic enough to join some sort of design system-package.